cypress/e2e/settings.cy.js

```diff
@@ -1,32 +1,85 @@
-/// <reference types="cypress" />
-/// <reference types="../support" />
+import homePageObject from '../support/pages/home.pageObject';
+import profilePageObject from '../support/pages/profile.pageObject';
+import settingsPageObject from '../support/pages/settings.pageObject';
```

Comment:
Use PageObject pattern for your tests:
create a files with POM classes for your pages in cypress/support/pages.
use PageObject.js file for the common for the whole app elements.

All these files should be added to the PR